cabal.project

```diff
@@ -40,8 +40,8 @@ source-repository-package
 
 source-repository-package
     type: git
-    location: https://github.com/kadena-io/chainweb-storage.git
-    tag: 1ee64539f9b6e27372caa14a1b4cbdf68736ed43
+    location: https://github.com/edmundnoble/chainweb-storage.git
+    tag: 838297f2885b256f5280b7f7ba3eea07ce1819cb
```

Comment:
I guess, this needs to be updated once https://github.com/kadena-io/chainweb-storage/pull/12 is merged?

Reply:
Yes.